BUILD.md

```diff
@@ -224,3 +224,12 @@ To start an installed development environment for a specific component, do:
 
 These installed images also generate .deb artifacts during build. They can be found
 in the `/tmp` dir in any of the images.
+
```

Comment:
This should be moved to README.md, similar to what we have in sawtooth-core repo.